src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.html

```diff
@@ -13,13 +13,13 @@ <h1 class="govuk-heading-l">Hearing details</h1>
       <br>
       <h1 class="govuk-heading-m">Hearing stage and result</h1>
       <span class="govuk-body align-right">
-        <a href="javascript:void(0)" [routerLink]="['/', 'hearings', 'actuals', 'hearing-stage-result']"
+        <a href="javascript:void(0)" [routerLink]="['/', 'hearings', 'actuals', id, 'hearing-stage-result']"
            class="govuk-link">Update</a>
       </span>
       <dl class="govuk-summary-list">
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">Hearing stage</dt>
-          <dd class="govuk-summary-list__value">{{hearingOutcome.hearingType}}</dd>
+          <dd class="govuk-summary-list__value" [innerHTML]="getHearingTypeDescription(hearingOutcome.hearingType)"></dd>
```

Comment:
avoid binding properties with method, you could define a variable in the ts and assign the variable here.